packages/sanity/src/desk/i18n/resources.ts

```diff
@@ -165,6 +165,35 @@ const deskLocaleStrings = {
 
   /** --- "PRODUCTION PREVIEW", eg link to content --- */
   'production-preview.menu-item.title': 'Open preview',
+
+  /** -- DESK TOOL -- */
```

Comment:
These are all already within the desk namespace, so do not need to be prefixed with `desk-tool`.